specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/networkSecurityPerimeter.json

```diff
@@ -0,0 +1,416 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-01",
+    "title": "StorageManagementClient",
+    "description": "The Azure Storage Management API."
+  },
+  "paths": {
```

Comment:
Here missed several properties like "host", "schemes","consumes","produces"? 
Why we missed them?

Please refer storage swagger to add it: https://github.com/Elvis-Shi/azure-rest-api-specs/blob/69bea7edf1527d0947b7aabd82db8fa75132353a/specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/storage.json#L8
Or some SDK/tools generation from swagger will fail.

Reply:
Thanks. Fixed.